apps/src/templates/VisualizationOverlay.js

```diff
@@ -3,8 +3,8 @@
 
 var constants = require('../constants');
 var CrosshairOverlay = require('./CrosshairOverlay');
-var gridUtils = require('./gridUtils');
-var elementUtils = require('./designElements/elementUtils');
+var gridUtils = require('../applab/gridUtils');
+var elementUtils = require('../applab/designElements/elementUtils');
```

Comment:
Uhh... er... yeah.

To fix this I need to figure out a clean way to break out / inject the 'dragging' and 'element ID' behaviors for the overlay.  Or make an `AppLabVisualizationOverlay` and a `GameLabVisualizationOverlay`.  I'd prefer the former.